core/src/main/java/io/grpc/CallCredentials.java

```diff
@@ -92,16 +100,16 @@ void applyRequestMetadata(
    * <p>Exactly one of its methods must be called to make the RPC proceed.
    */
   @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1914")
-  public interface MetadataApplier {
+  public abstract class MetadataApplier {
```

Comment:
static?

Reply:
An class under an interface is implicitly static. But it's no harm to add it (and it will be needed once CallCredentials becomes an abstract class).